python/thirdweb-ai/src/thirdweb_ai/services/insight.py

```
@@ -409,3 +418,163 @@ def decode_signature(
+            for agg in out["aggregations"]:
+                if isinstance(agg, dict):
+                    for value in agg.values():
+                        if isinstance(value, dict) and "chain_id" in value:
+                            value.pop("chain_id")
+
+        return out
+
+    @tool(
+        description="Aggregate blockchain transactions with powerful grouping and aggregation options. Use this for transaction counts, volumes, sums, and other analytics grouped by address, block, or time period."
+    )
+    def aggregate_transactions(
```

Pylint (3.3.7)

[refactor] Too many arguments (10/5)

(R0913)